chemicaltoolbox/pgchem/pgchem/db_filter.xml

```diff
@@ -120,15 +112,22 @@
               <param name="rings_min" type="integer" value="" label="Minimum number of rings"/>
               <param name="rings_max" type="integer" value="" label="Maximum number of rings"/>
             </when>
+          
           </conditional>
         </repeat>
       </when>
+      <when value="FragmentLike"/>
+      <when value="DrugLike"/>  
+      <when value="LeadLike"/>  
     </conditional>
   </inputs>
   <outputs>
     <!-- Only fetch smiles from the database since SDF retrieval takes ages -->
     <data format="smi" name="outfile"/>
   </outputs>
+  <tests>
+    <test expect_failure="true" />
```

Comment:
lets not do this, rather not push the tool to the TS ... rename the .shed.yml to something else and we need to come back and fix this tool if there is a use-case ...